app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -1,14 +1,7 @@
 import * as yup from "yup";
 
-const CONTACTS_LIST = ["Oliver Smith", "Bilbo Baggins", "Frodo Baggins"];
-const TAGS_LIST = [
-  "Getting started",
-  "Onboarding",
-  "User Flow",
-  "UX",
-  "Bugs",
-  "V2",
-];
+const CONTACTS = ["Oliver Smith", "Bilbo Baggins", "Frodo Baggins"];
+const TAGS = ["Getting started", "Onboarding", "User Flow", "UX", "Bugs", "V2"];
 
 export const CONTACT_OPTIONS = [
```

Comment:
Create the options using the base entity. Here you already have `CONTACTS`. So you can create `CONTACT_OPTIONS` by iterating through the `CONTACTS`. Refer slack thread.

Reply:
Fixed.